generator.go

```diff
@@ -690,8 +691,22 @@ func main() {
 	}
 
 	var s metaSchema
-	if err = json.Unmarshal(file, &s); err != nil {
-		log.Fatalln("Error parsing JSON:", err)
+	if *rootNodeName != "" {
+
+		var sub map[string]metaSchema
+		if err = json.Unmarshal(file, &sub); err != nil {
+			log.Fatalln("Error parsing JSON:", err)
+		}
+		node, ok := sub[*rootNodeName]
+		if ok == false {
+			log.Fatalln(fmt.Sprintf("Error JSON node '%s' does not exists:", *rootNodeName), err)
```

Comment:
Instead of combining `log.Fatalln` and `fmt.Sprintf`, use `log.Fatalf`. Don't forget to add `\n` at the end of the format string.